.github/workflows/ci.yml

```
@@ -72,6 +72,9 @@ jobs:
           - python-version: "3.9"
             runs-on: ubuntu-latest
             cmake-version: "3.20.x"
+          - python-version: "pypy-3.10"
+            runs-on: macos-latest
+            cmake-version: "3.15.x"
```

Just for debugging, can you try first `pypy-3.9` to make sure that that is not available for `macos-latest`?

---

Nah, just bump it to `pypy-3.10` as the "minimum version we support". Thanks for the issue.

---

I'm not sure 3.15 supported macOS ARM, and people don't have old cmake on macOS ARM anyway.


```suggestion
          - python-version: "pypy-3.10"
            runs-on: macos-latest
            cmake-version: "4.0.x"
```